src/gui/CMakeLists.txt

```diff
@@ -259,3 +273,4 @@ if(INPUTLEAP_BUILD_TESTS)
         target_link_libraries(guiunittests Qt::Core5Compat)
     endif()
 endif()
+
```

Comment:
Unnecessary change.

Reply:
The file lacked a empty new line at the eof

Reply:
Now it's two newlines :)